### PR TITLE
bump es to version 2.4.4

### PR DESCRIPTION
**Pull request by richm**
@jcantrill @ewolinetz ptal
[test]

### CODE REVIEW

File: elasticsearch/Dockerfile [hunk 1]
```diff
@@ -8,10 +8,10 @@ USER 0
...
   JAVA_VER=1.8.0 \
-  ES_VER=2.4.1 \
-  SG_VER=2.4.1.8 \
-  SG_SSL_VER=2.4.1.16 \
-  ES_CLOUD_K8S_VER=2.4.1 \
+  ES_VER=2.4.4 \
+  SG_VER=2.4.4.10 \
+  SG_SSL_VER=2.4.4.19 \
+  ES_CLOUD_K8S_VER=2.4.4 \
   OSE_ES_VER=2.4.1.2 \
```
Comment (ewolinetz) [inline elasticsearch/Dockerfile]: Doesn't this need a version bump as well?

Comment (richm) [inline elasticsearch/Dockerfile]: Apparently not.  It seems to work just fine.

Comment (ewolinetz) [inline elasticsearch/Dockerfile]: I'm surprised that it installs then... the plugins specify the version of ES that they need to work with (big pain with the 2.x ES plugins)

Comment (ewolinetz) [inline elasticsearch/Dockerfile]: Yeah, look at the travis job, you can see that the ES plugin fails to install.

**Comment by richm** [comment]
@ewolinetz yep - I'm working on the 2.4.4.x release now

**Comment by richm** [comment]
@ewolinetz @jcantrill Does this look ok?

File: elasticsearch/sgconfig/sg_config.yml [hunk 2]
```diff
@@ -3,8 +3,7 @@ searchguard:
     http:
       xff:
         enabled: true
-        remoteIpHeader: 'remoteAddress'
-        proxiesHeader: 'remoteAddress'
+        remoteIpHeader: 'x-forwarded-for'
```
Comment (ewolinetz) [inline elasticsearch/sgconfig/sg_config.yml]: did we change this information coming from the kibana proxy?

Comment (richm) [inline elasticsearch/sgconfig/sg_config.yml]: No.  Basically, until SG 2.4, the above settings were ignored, so SG was always using the default, which was `x-forwarded-for`.  See https://github.com/floragunncom/search-guard/commit/056d14a64b88fae2484108e285d024bedd4f2e71

Comment (ewolinetz) [inline elasticsearch/sgconfig/sg_config.yml]: hmm, ack

**Comment by ewolinetz** [comment]
LGTM

**Comment by richm** [comment]
[merge]

**Pull request merged**